.travis.yml

```diff
@@ -69,6 +71,36 @@ matrix:
      script:
       - RUSTFLAGS="-D warnings" cargo build --release --verbose
       - RUSTFLAGS="-D warnings" cargo test --verbose
+   - language: android
+     dist: trusty
+     android:
+       components:
+        - build-tools-26.0.1
+        # Minimum API level supported
+        - android-21
+     install:
+      # Install rust manually
+      - curl https://build.travis-ci.org/files/rustup-init.sh -sSf | sh -s -- -y --default-toolchain stable
+      - export PATH=$HOME/.cargo/bin:$PATH
+      - rustup default stable
+      - rustup target add aarch64-linux-android arm-linux-androideabi armv7-linux-androideabi i686-linux-android
+      # Install android ndk
+      - NDK_VER=r13b
+      - curl -ondk.zip -q https://dl.google.com/android/repository/android-ndk-$NDK_VER-linux-x86_64.zip
+      - unzip -q ndk.zip -d $HOME
+      - export ANDROID_NDK_HOME=$HOME/android-ndk-$NDK_VER
+      # Install latest cmake. preinstalled version is too old (3.2.2)
+      - CMAKE_URL=https://cmake.org/files/v3.15/cmake-3.15.4-Linux-x86_64.tar.gz
```

Comment:
What version of cmake is required? The ubuntu bionic image has 3.12 https://docs.travis-ci.com/user/reference/bionic/#compilers-and-build-toolchain if we could just use that image we wouldn't need to install cmake manually.

Reply:
We need cmake 3.6.0 and later otherwise boringssl build will fail. For Android build we need to use `trusty` image. https://docs.travis-ci.com/user/languages/android/
And `trusty` image has cmake 3.2.2 (`/usr/bin/cmake`) so had to install newer version. This is outside of NDK build, so NDK cmake doesn't help.

Reply:
Ok. I was able to install `cmake` using `sdkmanager`, replacing this installation procedure.